docs/code_navigation/references/auto_indexing_configuration.mdx

```diff
@@ -63,15 +63,15 @@ List of environment variables needed. These are made accessible to steps, local_
 
 ### Examples
 
-The following example uses the Docker image `sourcegraph/lsif-go` pinned at the tag `v1.6.7` and additionally secured with an image digest. This index configuration runs the Go indexer with quiet output in the `dev/sg` directory and uploads the resulting index file (`dump.lsif` by default).
+The following example uses the Docker image `sourcegraph/scip-go` pinned at the tag `v0.1` and additionally secured with an image digest. This index configuration runs the Go indexer with quiet output in the `dev/sg` directory and uploads the resulting index file (`index.scip` by default).
 
 ```json
 {
-  "indexer": "sourcegraph/lsif-go:v1.6.7@sha256::9c2d9cf...1baed2b",
+  "indexer": "sourcegraph/scip-go:v0.1@sha256::39c1495...55391cc",
   "indexer_args": [
-    "lsif-go",
+    "scip-go",
     "-q",
-    "--no-animation"
+    "--animation"
```

Comment:
```suggestion
```

Animation is turned off by default, there was a bug in the help text which I just fixed. https://github.com/sourcegraph/scip-go/pull/83